docs/api-docs/storefront/graphql/graphql-product-reviews.mdx

```diff
@@ -0,0 +1,191 @@
+
+| UI Name | Permission | Parameter |
+|:--------|:-----------|:----------|
+| Content | modify | `store_v2_content` |
+| Content | read-only | `store_v2_content_read_only` |
+
+## Store settings
+
+There are a few store settings to consider when creating a product review:
+
+- [Overall enable product reviews](https://support.bigcommerce.com/s/article/Managing-Reviews?language=en_US#enable)
+- [Only allow reviews from previous customers](https://support.bigcommerce.com/s/article/Managing-Reviews?language=en_US#enable)
+- [Enable reCAPTCHA](https://support.bigcommerce.com/s/article/Updating-Themes-with-reCAPTCHA-v2)
+
+**Overall enable product reviews**
+
+The merchant must enable this overall setting for the review submission process to work. Disabling this setting cancels any attempts to submit reviews.
+
+**Only allow reviews from previous customers**
+
+When the merchant enables the 'Only allow reviews from previous customers' setting, only logged-in customers (not guest shoppers) can submit a review, and they must have at least one previous order placed in the store.
+This setting ensures the merchant has a genuine customer to contact if needed. You can use the [login mutation](/docs/storefront/graphql#customer-login) to log in a customer.
+Once a purchase has been made, customers can review any item in the store. The order doesn't have to include the product being reviewed. 
+
+When the merchant disables the setting, guest shoppers can also submit reviews, but they must provide an email. Since guests are not logged in, they must supply the email in the request as an input parameter.
+Logged-in customers don't need to provide the email through the input parameter, as the system will use the email from the customer's account.
+If the logged-in customer provides an email as an input, BigCommerce will validate the email against the email associated with the shopper's account. If the email doesn't match, the submission will fail.
+
+In all cases, you can create only one review per product an email address.
+
+**Enable reCAPTCHA**
+
+If a merchant enables the [reCAPTCHA setting](https://support.bigcommerce.com/s/article/Updating-Themes-with-reCAPTCHA-v2) on the storefront, you need to send a reCAPTCHA token with the request.
+
+## Mutation
+
+The following mutation creates a product review for a store.
+
+
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```graphql filename="Example request: Create a product review" showLineNumbers copy
+mutation {
+  catalog {
+    addProductReview (
+      reCaptchaV2: "12345"
+      input: {
+        productEntityId: 81
+          review: {
+            title: "teitle"
+            text: "textr"
+            author: "oman"
+            rating: 4  # integer value between 1 and 5
+            email: "mem@ail.com2"
```

Comment:
Up to you, but maybe use some more "professional" fake data here. I know I gave this to you but I just threw in some random text. Do y'all have any consistent practice for fake data utilized in example input?

```
            title: "review title"
            text: "review body"
            author: "Author Name"
            rating: 4  # integer value between 1 and 5
            email: "emailOfReviewer@myemail.com"
```

Reply:
Likewise don't forget to update the output if you do change this.